ui/src/message_popup/popup_question_test.jsx

```diff
@@ -0,0 +1,47 @@
+/* flow weak */
+import React from 'react';
+
+import {shallow} from 'enzyme';
+import sinon from 'sinon';
+import {expect} from 'chai';
+import * as TestFixtures from './test_fixtures.js';
+
+import PopupQuestion from './popup_question.jsx';
+import SummaryCard from './summary_card.jsx';
+import ScenarioRenderer from './renderers/scenario_renderer.jsx';
+import PromptsRenderer from './renderers/prompts_renderer.jsx';
+import ResponseRenderer from './renderers/response_renderer.jsx';
+
+
+function testProps(props) {
```

Comment:
This is a common pattern - creating props for a component, and allowing variations on this to be passed in.